src/pages/settings/Wallet/ExpensifyCardPage.js

```diff
@@ -0,0 +1,102 @@
+import ONYXKEYS from '../../../ONYXKEYS';
+import ROUTES from '../../../ROUTES';
+import NotFoundPage from '../../ErrorPage/NotFoundPage';
+import CardPreview from '../../../components/CardPreview';
+import HeaderWithBackButton from '../../../components/HeaderWithBackButton';
+import MenuItemWithTopDescription from '../../../components/MenuItemWithTopDescription';
+import ScreenWrapper from '../../../components/ScreenWrapper';
+import assignedCardPropTypes from './assignedCardPropTypes';
+import useLocalize from '../../../hooks/useLocalize';
+import * as CurrencyUtils from '../../../libs/CurrencyUtils';
+import Navigation from '../../../libs/Navigation/Navigation';
+import styles from '../../../styles/styles';
+import * as CardUtils from '../../../libs/CardUtils';
+
+const propTypes = {
+    /* Onyx Props */
+    cardList: PropTypes.objectOf(assignedCardPropTypes),
+
+    /** Navigation route context info provided by react navigation */
+    route: PropTypes.shape({
+        params: PropTypes.shape({
+            /** domain passed via route /settings/wallet/card/:domain */
+            domain: PropTypes.string,
+        }),
+    }).isRequired,
+};
+
+const defaultProps = {
+    cardList: {},
+};
+
+function ExpensifyCardPage({
+    cardList,
+    route: {
+        params: {domain},
+    },
+}) {
+    const {translate} = useLocalize();
+    const domainCards = CardUtils.getDomainCards(cardList)[domain];
+    const virtualCard = _.find(domainCards, (card) => card.isVirtual) || {};
+    const physicalCard = _.find(domainCards, (card) => !card.isVirtual) || {};
+
+    if (_.isEmpty(virtualCard) || _.isEmpty(physicalCard)) {
+        return <NotFoundPage />;
+    }
+
+    const formattedAvailableSpendAmount = CurrencyUtils.convertToDisplayString(physicalCard.availableSpend || virtualCard.availableSpend || 0);
+
+    return (
+        <ScreenWrapper includeSafeAreaPaddingBottom={false}>
+            {({safeAreaPaddingBottomStyle}) => (
+                <>
+                    <HeaderWithBackButton
+                        title={translate('cardPage.expensifyCard')}
+                        onBackButtonPress={() => Navigation.goBack(ROUTES.SETTINGS_WALLET)}
+                    />
+                    <ScrollView contentContainerStyle={safeAreaPaddingBottomStyle}>
+                        <View style={[styles.flex1, styles.mb4, styles.mt4]}>
+                            <CardPreview />
+                        </View>
+
+                        <MenuItemWithTopDescription
+                            description={translate('cardPage.availableSpend')}
+                            title={formattedAvailableSpendAmount}
+                            interactive={false}
+                            titleStyle={styles.newKansasLarge}
+                        />
+                        <MenuItemWithTopDescription
+                            description={translate('cardPage.virtualCardNumber')}
+                            title={virtualCard.maskedPan}
```

Comment:
I brought this up in a comment, but we may create a helper function to format the maskedPAN into `•••• •••• •••• 1234` format depending on BE updates. I'll keep you updated on decisions